test_scripts/API/SetGlobalProperties/026_ATF_UI_IsReady_no_resp_SetGlobalProp_UI_SetGlobalProp_UNSUP_RES.lua

```diff
@@ -0,0 +1,161 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
```

Comment:
@istoimenova , script was updated in commit 9e46219c5c10e19aa72a3cb92276079989e650fe